nym-wallet/src/components/Delegation/DelegateModal.tsx

```diff
@@ -211,7 +211,7 @@ export const DelegateModal: React.FC<{
           initialValue={identityKey}
           readOnly={Boolean(initialIdentityKey)}
           textFieldProps={{
-            autoFocus: !initialIdentityKey,
+            focused: !initialIdentityKey,
```

Comment:
note sure why this changed? The autofocus doesnt seem to work now.

When the delegate modal opens the `Node identity key` field should focus`

Reply:
@fmtabbara I had some issues with the autofocus, but now I'm testing it, and seems ok. I'm going to revert this change, and keep it in mind if I see something weird in the future. Thanks